filefinder/tests/test_filefinder.py

```diff
@@ -480,6 +480,30 @@ def test_find_unparsable():
     ):
         ff.find_files()
 
+    with pytest.warns(match="Could not parse 'a/b' with the pattern '{cat}/{cat}'"):
+        ff.find_files(on_parse_error="warn")
+
+    expected = pd.DataFrame(list(), columns=["filename", "cat"])
+    result = ff.find_files(on_parse_error="skip")
+    pd.testing.assert_frame_equal(result.df, expected)
+
+    ff = FileFinder("{cat}", "{cat}", test_paths=["a/b", "a/a"])
+    expected = {"filename": {0: "a/a"}, "cat": {0: "a"}}
+    expected = pd.DataFrame.from_dict(expected)
+    result = ff.find_files(on_parse_error="skip")
+    pd.testing.assert_frame_equal(result.df, expected)
+
+    result = ff.find_files(on_parse_error="warn")
```

Comment:
This will result in a warning in the test - best to also wrap that in a `pytest.warns` block.